tests/test_github_interrupted_sync_add_stream.py

```diff
@@ -0,0 +1,178 @@
+from tap_tester import connections, runner, menagerie
+from base import TestGithubBase
+
+
+class TestGithubInterruptedSyncAddStream(TestGithubBase):
+    """Test tap's ability to recover from an interrupted sync"""
+
+    @staticmethod
+    def name():
+        return "tap_tester_github_interrupted_sync_add_stream_test"
+
+    def get_properties(self):
+        """
+        Maintain states for start_date and end_date
+        """
+        return {
+            'start_date' : '2021-10-01T00:00:00Z',
+            'repository': 'singer-io/test-repo singer-io/singer-python'
+        }
+
+    def test_run(self):
+        """
+        Testing that if a sync job is interrupted and state is saved with `currently_syncing`,
+        the next sync job kicks off and the tap picks back up on that `currently_syncing` stream.
```

Comment:
Update this comment with details about `currently_syncing_repo`.

Reply:
Added `currently_syncing_repo` details in doc string.